pytox/core.c

```diff
@@ -986,6 +986,30 @@ ToxCore_conference_peer_get_name(ToxCore* self, PyObject* args)
   return PYSTRING_FromString((const char*)buf);
 }
 
+static PyObject*
+ToxCore_conference_peer_get_public_key(ToxCore* self, PyObject* args)
+{
+  CHECK_TOX(self);
+
+  uint8_t buf[TOX_MAX_NAME_LENGTH];
+  memset(buf, 0, TOX_MAX_NAME_LENGTH);
```

Comment:
No need for memset. Every byte will be set by the tox function.

Reply:
even on errors?

Reply:
On errors, we don't touch buf again.

If we do touch it, there is a bug. If there was no error and any byte was not set, there is a bug. Both these bugs are caught in tests using ASAN, so it's valuable to leave buf uninitialised, otherwise those bugs would be masked.